website_docs/automatic_instrumentation.md

```diff
@@ -0,0 +1,389 @@
+
+### Interval metric reader
+
+| System property          | Environment variable     | Purpose                                                                      |
+|--------------------------|--------------------------|------------------------------------------------------------------------------|
+| otel.imr.export.interval | OTEL_IMR_EXPORT_INTERVAL | The interval in milliseconds between pushes to the exporter (default: 60000) |
+
+#### Customizing the OpenTelemetry SDK
+
+> This is highly advanced behavior and still in the prototyping phase. It may
+change drastically or be removed completely. Use with caution
+
+The OpenTelemetry API exposes SPI
+[hooks](https://github.com/open-telemetry/opentelemetry-java/blob/main/api/src/main/java/io/opentelemetry/trace/spi/TracerProviderFactory.java)
+for customizing its behavior, such as the `Resource` attached to spans or the
+`Sampler`.
+
+Because the auto instrumentation runs in a separate classpath than the
+instrumented application, it is not possible for customization in the
+application to take advantage of this customization. In order to provide such
+customization, you can provide the path to a JAR file including an SPI
+implementation using the system property `otel.initializer.jar`. Note that this
+JAR will need to shade the OpenTelemetry API in the same way as the agent does.
+The simplest way to do this is to use the same shading configuration as the
+agent from
+[here](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/cfade733b899a2f02cfec7033c6a1efd7c54fd8b/java-agent/java-agent.gradle#L39).
+In addition, you will have to specify the
+`io.opentelemetry.javaagent.shaded.io.opentelemetry.api.trace.spi.TraceProvider`
+to the name of the class that implements the SPI.
+
+## Manual Instrumentation
+
+> Starting with 0.6.0, and prior to version 1.0.0, `opentelemetry-javaagent-all.jar`
+only supports manual instrumentation using the `opentelemetry-api` version with the same version
+number as the Java agent you are using. Starting with 1.0.0, the Java agent will start supporting
+multiple (1.0.0+) versions of `opentelemetry-api`.
+
+With auto-instrumentation it may be desirable to also add manual
+instrumentation. In this scenario, it is critical that both automatic and
+manual instrumentation stitch together properly. Two options exist to configure
+manual instrumentation with the appropriate span context:
+
+- `@WithSpan` annotation: Simply annotate the functions or methods you wish to instrument
+- `getTracer`: Traditional way to instrument with OpenTelemetry
+
+Both options require a dependency on the `opentelemetry-api` library to get started.
+
+### Dependency
+
+#### Maven
+
+```xml
+  <dependencies>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-api</artifactId>
+      <version>0.10.0</version>
```

Comment:
this is very out of date. we should be recommending use of the pom, and, of course, version 1.0.1